pkgs/development/libraries/discord-gamesdk/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchzip
+, pkg-config
+, autoPatchelfHook
+}:
+
+in stdenv.mkDerivation rec {
+  pname = "discord-gamesdk";
+  version = "3.2.1";
+
+  src = fetchzip {
+    url = "https://dl-game-sdk.discordapp.net/${version}/discord_game_sdk.zip";
+    sha256 = "sha256-83DgL9y3lHLLJ8vgL3EOVk2Tjcue64N+iuDj/UpSdLc=";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  installPhase =
+    ''runHook preInstall
+    mkdir -p $out/lib
+    install -Dm555 lib/${stdenv.hostPlatform.uname.processor}/discord_game_sdk${stdenv.hostPlatform.extensions.sharedLibrary} $out/lib/discord_game_sdk${stdenv.hostPlatform.extensions.sharedLibrary}
+    runHook postInstall'';
```

Let's keep the lines shorter.
```suggestion
  installPhase =
    let
      processor = stdenv.hostPlatform.uname.processor;
      sharedLibrary = stdenv.hostPlatform.extensions.sharedLibrary;
    in
    ''
      runHook preInstall

      install -Dm555 lib/${processor}/discord_game_sdk${sharedLibrary} $out/lib/discord_game_sdk${sharedLibrary}

      runHook postInstall
    '';
```

---

Applied in latest push.